app/Order.php

```diff
@@ -54,6 +54,32 @@ public function user()
     }
 
     /**
+     * Update Payment
+     *
+     * @param OrderItem $orderItem It is object order item.
+     * @param int       $quantity  It is quantity change.
+     *
+     * @return mixed
+     */
+    public function updatePayment(OrderItem $orderItem, $quantity)
```

Comment:
hàm này để dùng chung được cho nhiều chỗ thì ko nên truyền `$orderItem` hay `$quantity` vào. Ở đây là cập nhật lại `total_price` cho order thôi, không cần quan tâm đến có sự kiện gì xảy ra đối với orderItem, vậy thì mới có thể dùng chung được cho các thay đổi ở orderItem
- có orderItem mới
- xoá  orderItem
- orderItem update quantity

Nghĩa là sum toàn bộ price của các orderItem hiện tại trong DB và update vào order.